index.js

```diff
@@ -152,7 +152,20 @@ function redisStore(args) {
       if (err) {
         return cb && cb(err);
       }
-      conn.del(key, handleResponse(conn, cb));
+
+      var args = []
+
+      if ( Object.prototype.toString.call( key ) === '[object Array]' ) {
```

Comment:
Can you use Array.isArray here instead of string compare ? http://web.mit.edu/jwalden/www/isArray.html